packages/client/src/entities/ForumTopic/ui/ForumTopicCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { PropsWithChildren } from 'react';
+import {
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+  CardHeader,
+  Avatar,
+} from '@mui/material';
+
+import { IForumTopic } from '@/shared/types';
+
+export type ForumTopicCardProps = {
+  author: IForumTopic['author'];
+  createAt: IForumTopic['createAt'];
+  title: IForumTopic['title'];
+  actions?: React.ReactNode;
+  body?: string;
+};
```

Comment:
Тут можно чуть-чуть покрасивее сделать:
```suggestion
export type ForumTopicCardProps = Pick<IForumTopic, 'author' | 'createAt' | 'title'> & {
  actions?: React.ReactNode;
  body?: string;
};
```